Edits to pgd pr5104 (#5299)
EnterpriseDB/docs: product_docs/docs/pgd/5/admin-manual/installing/08-using-pgd-cli.mdx

EOF
```

You can repeat this process on any system where you need to use PGD CLI.

### Run PGD CLI

With the configuration file in place, and logged in as the enterprisedb system user, you can run pgd-cli. 

> For example, yuo can use the `show-nodes` command to list the nodes in yur cluster and their status:

Reviewer: ```suggestion
For example, you can use the `show-nodes` command to list the nodes in your cluster and their status:
```
